packages/did/src/DidDocumentExporter/DidDocumentExporter.spec.ts

```diff
@@ -239,46 +273,74 @@ describe('Light DID Document exporting tests', () => {
     type: 'ed25519',
   }
   let encryptionDidKeyDetails: INewPublicKey | undefined
+  let serviceEndpoints: IDidServiceEndpoint[]
 
-  it('exports the expected application/json W3C DID Document with an Ed25519 authentication key, and an x25519 encryption key', () => {
+  it('exports the expected application/json W3C DID Document with an Ed25519 authentication key, an x25519 encryption key, and some service endpoints', () => {
```

Comment:
```suggestion
  it('exports the expected application/json W3C DID Document with an Ed25519 authentication key, an x25519 encryption key, and two service endpoints', () => {
```

Reply:
Keeping it consistent.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/428/commits/2728528fd48c996216b1a40334b63042706089ef